### ISSUE TITLE
Only one augmentation method

### ISSUE THREAD

**Issue by VCasecnikovs** [zylo117/Yet-Another-EfficientDet-Pytorch#180]
Hello, I found out that there is only x_flip augmenter as it was in a paper. Do you think that this is enough? I think we can try to add optional y_flip to satellite data e.t.c I even think about mixup, but I haven't seen nice realization on object detection data. Thank you for your work. I'm having fun reading your code.)

**Comment by zylo117** [comment]
Hi, h_flipping is proved to be useful in most of the detection network, while others augmentations don't always help. Of courese, you're welcome to apply some augmentations though. But I tried using more augmentations before but it ended up increase both training and val loss.
